{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "e35986cb_2249c2a6",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 2
      },
      "lineNbr": 0,
      "author": {
        "id": 14134
      },
      "writtenOn": "2023-03-13T17:31:09Z",
      "side": 1,
      "message": "It looks like this patch is failing in the CI because of a missing shared library.  Is it part of the standard bionic install?\n\nhttps://chromium-swarm.appspot.com/task?id\u003d60f476219fff4f10\n\n```\nAdditional test environment:\n    CHROME_HEADLESS\u003d1\n    GTEST_SHARD_INDEX\u003d0\n    GTEST_TOTAL_SHARDS\u003d6\n    LANG\u003den_US.UTF-8\nCommand: /b/s/w/ir/.task_template_vpython_cache/vpython/e05872/bin/python ../../tools_webrtc/gtest-parallel-wrapper.py --output_dir\u003d/b/s/w/iowvr0j2u_/test_logs --gtest_color\u003dno --retry_failed\u003d3 ./modules_unittests --isolated-script-test-output\u003d/b/s/w/iowvr0j2u_/output.json --isolated-script-test-perf-output\u003d/b/s/w/iowvr0j2u_/perftest-output.json\n\ngtest-parallel-wrapper: Executing command /b/s/w/ir/.task_template_vpython_cache/vpython/e05872/bin/python /b/s/w/ir/third_party/gtest-parallel/gtest-parallel --shard_index\u003d0 --shard_count\u003d6 --output_dir\u003d/b/s/w/iowvr0j2u_/test_logs --dump_json_test_results\u003d/b/s/w/iowvr0j2u_/output.json --retry_failed\u003d3 --gtest_color\u003dno ./modules_unittests -- --isolated-script-test-perf-output\u003d/b/s/w/iowvr0j2u_/perftest-output.json\n./modules_unittests: Command \u0027[\u0027./modules_unittests\u0027, \u0027--isolated-script-test-perf-output\u003d/b/s/w/iowvr0j2u_/perftest-output.json\u0027, \u0027--gtest_list_tests\u0027]\u0027 returned non-zero exit status 127.\nb\u0027./modules_unittests: error while loading shared libraries: libgdk-3.so.0: cannot open shared object file: No such file or directory\\n\u0027\n```",
      "revId": "89c0694f35b5ffe229ef80cd5d9e80a8623781e3",
      "serverId": "58829da1-049c-39fb-b951-ebdcd0984767"
    }
  ]
}